connectivity/FEATURE_BLE/source/generic/GapImpl.h

```diff
@@ -351,6 +351,223 @@ class Gap :
     /* ===================================================================== */
     /*                    private implementation follows                     */
 
+private:
+    /** Last in First Out list*/
```

Comment:
OK, that's wrong I'll fix that, but that won't affect the rest of the code

Reply:
Actually, it only matters for connections and since there can only be one connection to each peer there's no difference